package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "memoize-fs",
   "version": "2.0.0",
-  "description": "memoize/cache in file system solution for Node.js",
+  "description": "Node.js solution for memoizing/caching a function and its return state into the file system",
   "author": "Boris Diakur <contact@borisdiakur.com> (https://github.com/borisdiakur)",
   "scripts": {
-    "test": "npm run eslint && mocha test",
-    "eslint": "eslint .",
-    "mocha": "mocha --reporter nyan test",
+    "lint": "eslint .",
+    "pretest": "npm run lint",
+    "test": "mocha --reporter nyan test",
```

Comment:
I’d rather keep linting in the test script as I would like linting to be executed in ci.

Reply:
It still will be. It's called in the `pretest` which is always called just before `test`.

Reply:
You are right, I have overlooked the pretest script. 👍